### PR TITLE
Pl support sbml v1.0

### PR DESCRIPTION
**Pull request by paulflang**

### CODE REVIEW

File: Manifest.toml [hunk 1]
```diff
@@ -0,0 +1,1162 @@
+# This file is machine-generated - editing it directly is not advised
+
+[[AbstractAlgebra]]
```
Comment (ChrisRackauckas) [inline Manifest.toml:3]: We shouldn't include a Manifest

Comment (paulflang) [reply Manifest.toml:3]: Reason I added the Manifest this time was cause I want to use SBML#master for CI, which is several commits ahead of https://github.com/LCSB-BioCore/SBML.jl/tree/v0.10.1
Got me really confused yesterday.

Comment (paulflang) [reply Manifest.toml:3]: I forgot the customs for version updating. How to deal with this? Once Mirek bumps the SBML version, we

1. bump the SBML compat
2. bump the SBMLtk patch version
3. remove Manifest
4. merge with main?

**Comment by paulflang** [comment]
Deprecated via #71 .

**Pull request closed without merging**